src/components/base/accordion/helpers/calcHeights.ts

```diff
@@ -0,0 +1,26 @@
+import { AccordionElement } from '../types';
+
+export const calcHeights = (items: AccordionElement[], resize: boolean) => {
+	if (!items) {
+		return;
+	}
+
+	items.forEach(obj => {
+		const btn = obj.querySelector('[data-accordion-trigger]') as HTMLElement;
```

Comment:
jeśli `obj` to `AccordionElement` to możemy wyciągnać `_content` a niżej `_trigger`, ale niżej bedzie trzeba sprawdzić, czy te elementy istnieją